source/patterns/@aws-solutions-constructs/aws-apigatewayv2websocket-sqs/README.md

```diff
@@ -0,0 +1,129 @@
+# aws-apigateway-sqs module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Stable](https://img.shields.io/badge/cfn--resources-stable-success.svg?style=for-the-badge)
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_apigateway_sqs`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-apigateway-sqs`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.apigatewaysqs`|
+
+## Overview
+
+This AWS Solutions Construct implements an Amazon API Gateway connected to an Amazon SQS queue pattern.
+
+Here is a minimal deployable pattern definition:
+
+Typescript
+``` typescript
+import { Construct } from 'constructs';
+import { Stack, StackProps } from 'aws-cdk-lib';
+import { ApiGatewayToSqs, ApiGatewayToSqsProps } from "@aws-solutions-constructs/aws-apigateway-sqs";
+
+new ApiGatewayToSqs(this, 'ApiGatewayToSqsPattern', {});
+```
+
+Python
+``` python
+from aws_solutions_constructs.aws_apigateway_sqs import ApiGatewayToSqs
+from aws_cdk import Stack
+from constructs import Construct
+
+ApiGatewayToSqs(self, 'ApiGatewayToSqsPattern')
+```
+
+Java
+``` java
+import software.constructs.Construct;
+
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awsconstructs.services.apigatewaysqs.*;
+
+new ApiGatewayToSqs(this, "ApiGatewayToSqsPattern", new ApiGatewayToSqsProps.Builder()
+        .build());
+```
+
+## Pattern Construct Props
+
+| **Name**     | **Type**        | **Description** |
+|:-------------|:----------------|-----------------|
+|apiGatewayProps?|[`api.RestApiProps`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_apigateway.RestApiProps.html)|Optional user-provided props to override the default props for the API Gateway.|
```

Comment:
this should be `webSocketApiProps` and link to said props class ?

Reply:
Maybe let's just do a close look over all the props in the readme to ensure they match what the index.ts file for the construct exposes. I was going to leave another comment to verify that `existingWebSocketApi` also looks to be missing.

Reply:
Updated.